next.config.js

```diff
@@ -1,4 +1,11 @@
 /** @type {import('next').NextConfig} */
-const nextConfig = {}
-
-module.exports = nextConfig
+module.exports = {
+    webpack(config) {
+      config.module.rules.push({
+        test: /\.svg$/,
+        use: ["@svgr/webpack"]
+      });
+  
+      return config;
+    }
```

Comment:
Out of ignorance...what do we need this for? Is this optimizing the svgs?

Reply:
Indeed Next.js doesn't allow to import SVGs, it encourage to use its Image component instead.